com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -707,6 +667,38 @@ private bool ProcessEvent()
         private void Update()
         {
             if (IsDisabledBySimulator)
+            {
+                // Flush all send queue items
+                foreach (var queueItem in m_SendQueue)
+                {
+                    queueItem.Value.Consume(queueItem.Value.Length);
+                }
+
+                // Flush all connection requests
+                while (m_Driver.IsCreated)
```

Comment:
Why is this looping over the driver creation status? Nothing in the loop can change it.

Reply:
Just kinda like an infinite while loop with an if condition. Nothing in the loop can change it, but I want to loop only if the driver is created, and then I break out of it in the loop